assets/app/scripts/controllers/projects.js

```diff
@@ -24,6 +24,45 @@ angular.module('openshiftConsole')
       $('#openshift-logo').off('click.projectsPage');
     });
 
+    $scope.toggleDelete = function(project) {
```

Comment:
i actually use the project name (string) in deleteProject as well, so i'll just make them both use `projectName`